Imagine/Cache/Resolver/WebPathResolver.php

```diff
@@ -169,6 +169,25 @@ private function getFullPath($path, $filter)
         // crude way of sanitizing URL scheme ("protocol") part
         $path = str_replace('://', '---', $path);
 
+        // transliterate
+        $path = iconv('utf-8', 'us-ascii//TRANSLIT', $path);
+
+        // remove unwanted characters
+        $path = preg_replace('~[^-/.\w]+~', '', $path);
+
+        // trim
+        $path = trim($path, '-');
+
+        // remove duplicate -
+        $path = preg_replace('~-+~', '-', $path);
+
+        // lowercase
+        $path = strtolower($path);
```

Comment:
cs-fixer wants mb_strtolower. can we use that or is there a specific reason to not use the mb_ version?

Reply:
Hi,

Can't say for sure but from what I can tell mb_string sort of adds support for more characters such as, and I quote the doc, "A-umlaut (Ä)" (reminds me my german classes :p).
As we're talking about a path here, there shouldn't be any of these characters and, as he who can do more can do less, I don't see any reason why mb_string would raise any problem.

Reply:
yeah that makes sense. it is considered general best practice to use mb_ thats why the cs fixer is complaining. but for this place it indeed seems unnecessary. is there a php comment you can put above this line to have the cs fixer ignore the line?

Reply:
I am happy to update the code with the mb version. Not sure what is the comment I need to put otherwise though.

Reply:
don't know by heart either. lets use mb_strtolower. this also will not mess up if we somehow miss multibyte characters in the path (though it might mess up later in the process if the character is not supported...)